src/command_modules/azure-cli-feedback/azure/cli/command_modules/feedback/custom.py

```diff
@@ -46,6 +46,8 @@
 _EXTENSIONS_ISSUES_URL = "aka.ms/azcli/ext/issues"
 _RAW_EXTENSIONS_ISSUES_URL = "https://github.com/Azure/azure-cli-extensions/issues/new"
 
+_EXTENSION_REPOS_URL_TO_ISSUES_URL = '/issues/new'  
```

Comment:
maybe rename this to something like `new_issues_suffix` or `simply new_issues_str` and inline in the only function that uses it. What do you think?